Tests/SwiftDocCTests/Infrastructure/DocumentationContext+RootPageTests.swift

```diff
@@ -36,7 +36,7 @@ class DocumentationContext_RootPageTests: XCTestCase {
                 """),
                 InfoPlist(displayName: "TestBundle", identifier: "com.test.example"),
             ]),
-        ])
+        ], atRoot: createTemporaryDirectory().appendingPathComponent("tempFolder"))
```

Comment:
Mostly out of curiosity- do we need the additional `tempFolder` here?

Reply:
The `TempFolder` expected the parent directory to exist but the target directory to not exist. I've removed the `TempFolder` type in the latest changes.